express/code/blocks/print-product-detail/createComponents/createProductInfoHeadingSection.js

```diff
@@ -0,0 +1,131 @@
+import { getLibs } from '../../../scripts/utils.js';
+import { formatDeliveryEstimateDateRange, formatLargeNumberToK, formatPriceZazzle } from '../utilities/utility-functions.js';
+
+let createTag;
+
+function createProductTitle(productDetails) {
+  const productTitleContainer = createTag('div', { class: 'pdpx-product-title-container' });
+  const productTitle = createTag('h1', { class: 'pdpx-product-title', id: 'pdpx-product-title', 'data-skeleton': 'true' }, productDetails.productTitle);
+  productTitleContainer.appendChild(productTitle);
+  return productTitleContainer;
+}
+
+function createStarRatings(productDetails) {
+  const starRatings = createTag('div', { class: 'pdpx-star-ratings' });
+  for (let i = 0; i < 5; i += 1) {
+    const star = createTag('img', { class: 'pdpx-product-info-header-ratings-star', src: '/express/code/icons/star-sharp.svg' });
+    starRatings.appendChild(star);
+  }
+  return starRatings;
+}
+
+function createRatingsNumber(productDetails) {
+  const ratingsNumberText = Math.round(productDetails.averageRating * 10) / 10;
+  const ratingsNumberContainer = createTag('div', { class: 'pdpx-ratings-number-container' });
+  const ratingsNumber = createTag('span', { class: 'pdpx-ratings-number', id: 'pdpx-ratings-number' }, ratingsNumberText);
+  ratingsNumberContainer.appendChild(ratingsNumber);
+  return ratingsNumberContainer;
+}
+
+function createRatingsAmount(productDetails) {
+  const ratingsAmountText = formatLargeNumberToK(productDetails.totalReviews);
+  const ratingsAmountContainer = createTag('div', { class: 'pdpx-ratings-amount-container' });
+  ratingsAmountContainer.className = 'pdpx-ratings-amount-container';
+  const ratingsAmount = createTag('button', { class: 'pdpx-ratings-amount', id: 'pdpx-ratings-amount', type: 'button' }, ratingsAmountText);
+  ratingsAmountContainer.appendChild(ratingsAmount);
+  return ratingsAmountContainer;
+}
+
+function createProductRatingsLockup(productDetails) {
+  const productRatingsLockupContainer = createTag('div', { class: 'pdpx-product-ratings-lockup-container' });
+  const starRatings = createStarRatings(productDetails);
+  const ratingsNumber = createRatingsNumber(productDetails);
+  const ratingsAmount = createRatingsAmount(productDetails);
+  productRatingsLockupContainer.append(starRatings);
```

Comment:
in many cases, we can combine those append calls like `productRatingsLockupContainer.append(starRatings, ratingsNumber, ratingsAmount);` to reduce some bytes